apps/psp/next.config.js

```diff
@@ -1,5 +1,5 @@
 const withImages = require('next-images');
-const withTM = require('next-transpile-modules')(['@team21/ui-components']);
+const withTM = require('next-transpile-modules')(['@team21/ui-components', '@team21/web-shop-front']);
```

Comment:
if you moved pages to web-shop-front, this may be unnecessary

Reply:
Yeap 😺